Source/ARTPush.h

```diff
@@ -75,13 +77,17 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)deactivate;
 
+- (void)getDevicePushDetails:(ARTPushStateCallback)callback;
```

Comment:
I think this needs to be agreed cross-platform (i.e. via the features spec) before we add it here. I think https://github.com/ably/docs/issues/1288 would be the best place to come to a consensus on this.